ArjunaJTA/jdbc/tests/classes/com/hp/mwtests/ts/jdbc/basic/ConnectionImpleTest.java

```diff
@@ -125,13 +124,14 @@ public void beforeCompletion() { }
             public void afterCompletion(int status) {
                 try {
                     connectionToTest.isClosed();
+                    connectionToTest.close();
```

Comment:
Hi gytis, I don't think I am being clear here. The connection we would leak would be a ConnectionImpl.

But lets not talk hypothetical. If this test does not call connection.close() then the following line is not called:  https://github.com/tomjenkinson/narayana/blob/bcc7c01599e46fcd5e5412ce9277bda62146ddb0/ArjunaJTA/jdbc/classes/com/arjuna/ats/internal/jdbc/ConnectionImple.java#L388